homeassistant/components/unifi/sensor.py

```diff
@@ -352,6 +353,38 @@ class UnifiSensorEntityDescription(
         unique_id_fn=lambda hub, obj_id: f"password-{obj_id}",
         value_fn=lambda hub, obj: obj.x_passphrase,
     ),
+    UnifiSensorEntityDescription[Devices, Device](
+        key="Device CPU utilization",
+        device_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        native_unit_of_measurement=PERCENTAGE,
+        api_handler_fn=lambda api: api.devices,
+        available_fn=async_device_available_fn,
+        device_info_fn=async_device_device_info_fn,
+        name_fn=lambda device: "CPU utilization",
+        object_fn=lambda api, obj_id: api.devices[obj_id],
+        supported_fn=lambda hub, obj_id: hub.api.devices[obj_id].raw.get(
+            "system-stats", False
+        ),
+        unique_id_fn=lambda hub, obj_id: f"device_cpu_utilization-{obj_id}",
```

Comment:
I think you can skip device here, there are nothing else that can report cpu so I think cpu utilisation is name space enough :)
```suggestion
        unique_id_fn=lambda hub, obj_id: f"cpu_utilization-{obj_id}",
```